packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -105,15 +104,64 @@ + (NSString * _Nullable)version {
   return nil;
 }
 
+// Expo developers: Enable the below code by running
+//     export EX_DEV_LAUNCHER_URL=http://localhost:8090
+// before doing pod install. This will cause the controller to see if
+// the expo-launcher packager is running, and if so, use that instead of
+// the prebuild bundle.
+// See the pod_target_xcconfig definition in expo-dev-launcher.podspec
+
+- (NSURL * _Nullable)devLauncherURL
+{
+#ifdef EX_DEV_LAUNCHER_URL
+  NSString *urlString = [NSString stringWithFormat:@"%s/%s", EX_DEV_LAUNCHER_URL, EX_DEV_LAUNCHER_PACKAGER_PATH];
+  return [NSURL URLWithString:urlString];
+#endif
+  return nil;
+}
+
+- (NSURL * _Nullable)devLauncherStatusURL
```

Comment:
```suggestion
- (nullable NSURL *)devLauncherStatusURL
```

Reply:
Relatedly, these files seem not to use NS_ASSUME_NONNULL_BEGIN/END (maybe it's automatically specified through the Podspec somehow?) -- it's out of scope for this PR but in general we try to use nullability modifies where we can, especially to act as a bit of documentation and to make it easier to translate code to Swift.